spec/services/decisions/income_decision_tree_spec.rb

```diff
@@ -85,6 +85,27 @@
         allow(form_object).to receive_messages(income_above_threshold: YesNoAnswer::NO)
       end
 
+      it { is_expected.to have_destination(:frozen_income_savings_assets, :edit, id: crime_application) }
+    end
+  end
+
+  context 'when the step is `frozen_income_savings_assets`' do
+    let(:form_object) { double('FormObject') }
+    let(:step_name) { :frozen_income_savings_assets }
+
+    context 'when they have frozen income or assets' do
+      before do
+        allow(form_object).to receive_messages(has_frozen_income_or_assets: YesNoAnswer::YES)
+      end
+
+      it { is_expected.to have_destination('/home', :index, id: crime_application) }
+    end
+
+    context 'when they do not have fozen income or assets' do
```

Comment:
Not to be pedantic but another typo in frozen 😅

Reply:
what am I like 🤦 

No thank you for spotting!

Reply:
Have amended the commit and pushed a fix :)

Reply:
Happens to me all the time! I'm happy with pr aside from that